src/apps/external_executor/protobuf/misc.proto

```diff
@@ -1,7 +1,6 @@
 syntax = "proto3";
 
 import "google/protobuf/empty.proto";
-import "kv.proto";
 
 package temp;
```

Comment:
If this is intended to be an example, maybe we can rename this to `example`

Reply:
Yes. We will move the gRPC-specific endpoints to a new app altogether but I'll leave this to a further PR.